deny.toml

```diff
@@ -0,0 +1,33 @@
+[advisories]
+ignore = [
+    {id="RUSTSEC-2021-0139", reason="The maintainer has advised that this crate is deprecated and will not receive any maintenance. https://rustsec.org/advisories/RUSTSEC-2021-0139.html"},
```

Comment:
@insipx do we need to create an issue for this to decide about it? we need to user other alternatives.